etc/docs/preview.ts

```diff
@@ -0,0 +1,36 @@
+import { on } from 'events';
+import { PathLike } from 'fs';
+import { access, readFile } from 'fs/promises';
+import { createServer } from 'http';
+import { join } from 'path';
+
+const fromDir = (...filePath) => join('docs', ...filePath);
+const server = createServer();
+server.listen(8080, 'localhost', 128, () => console.log('Listening on http://localhost:8080'));
+const canAccess = async (pathlike: PathLike) => await access(pathlike).then(() => true,() => false);
+
+async function main() {
+  for await (const [request, response] of on(server, 'request')) {
+    const url = new URL(request.url, `http://${request.headers.host}`);
+    console.log(request.method, url.pathname);
+    const pathname = url.pathname.replace('/node-mongodb-native', '');
+    console.log(pathname)
```

Comment:
```suggestion
```

I think we should instead fix the paths in the html, we shouldn't have to do this request editing, but we need to double check on your test fork. All our src/img/css tags should start after the `/node-mongodb-native` part. 

This:
```html
<link rel="stylesheet" href="/node-mongodb-native/s/lib/bootstrap.min.css" type="text/css" />
```
Should be:
```html
<link rel="stylesheet" href="s/lib/bootstrap.min.css" type="text/css" />
```